embed/config.go

```diff
@@ -136,6 +136,13 @@ type Config struct {
 	//	}
 	//	embed.StartEtcd(cfg)
 	ServiceRegister func(*grpc.Server) `json:"-"`
+
+	// auth
+
+	TokenType      string
```

Comment:
I think it is a little bit difficult to determine the default. simple token should be avoided in production and jwt can be used in production but it requires additional keys. How about setting the default "" and let `etcdctl auth enable` cause failure in such a case? Users will be able to find that the auth related config is bad earlier.